### PR TITLE
unify flash message calls + indicator for unreviewed places

### PR DESCRIPTION
**Pull request by kjoscha**
wip

### CODE REVIEW

**Comment by kjoscha** [comment]
ready2merge

**Comment by kjoscha** [comment]
ready2merge

**Comment by kjoscha** [comment]
closes #580 
closes #581

File: app/controllers/places_controller.rb [hunk 1]
```diff
@@ -36,14 +36,16 @@ def update
       AttributeSetter::Place.set_attributes_after_update(place: @place, params: @params_to_commit, privileged: has_privileged_map_access)
       store_in_session_cookie
 
+      @place.reviewed ? message = 'Successfully updated!' : message = 'Successfully updated! Waiting for admin`s review...'
```
Comment (andneuma) [inline app/controllers/places_controller.rb]: Gleich ne Übersetzung einbauen!

File: app/controllers/places_controller.rb [hunk 2]
```diff
@@ -67,14 +69,16 @@ def create
       AttributeSetter::Place.set_attributes_after_create(place: @place, params: @params_to_commit, privileged: has_privileged_map_access)
       store_in_session_cookie
 
+      @place.reviewed ? message = 'Successfully created!' : message = 'Successfully created! Waiting for admin`s review...'
```
Comment (andneuma) [inline app/controllers/places_controller.rb]: Same

**Pull request merged**